README.md

```diff
@@ -25,7 +25,7 @@
 # 📖 Blog App <a name="about-project"></a>
```

Comment:
### Your Getting Started Section :red_circle:
![image](https://user-images.githubusercontent.com/93039359/215249644-8659a92a-c31e-44d3-8fad-1f8845d41cd1.png)
### Professional Getting Started Section :green_circle:
![image](https://user-images.githubusercontent.com/93039359/215249858-39f136e0-0f8a-49ed-9074-8369492ad2e6.png)

**[Optional]**

- [ ] Your Getting Started Section doesn't contain all the required commands to start your application :fearful: Kindly add all the commands to make your readme file descriptive and professional :+1: